packages/components/src/ConfirmationModal/ConfirmationModal.mdx

```diff
@@ -26,6 +26,8 @@ import {
 } from "@jobber/components/ConfirmationModal";
 ```
 
+Default type of confirmation Modal
```

Comment:
We don't need to say `default` as it is the first one on the page. It is assumed that it is the default.